### PR TITLE
config-template: add `server.max-grpc-send-msg-len` to template

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

### What problem does this PR solve?

Problem Summary:

`server.max-grpc_send_msg_len` was not shown in config template.

### What is changed and how it works?

What's Changed:
Add `server.max-grpc_send_msg_len` to template

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`: TODO
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test

### Release note 
* No release note.

### CODE REVIEW

File: etc/config-template.toml [hunk 1]
```diff
@@ -146,6 +146,9 @@
 ## Time to wait before closing the connection without receiving KeepAlive ping Ack.
 # grpc-keepalive-timeout = "3s"
 
+## Set maximum message length in bytes that gRPC can send. `-1` means unlimited.
+# max-grpc_send_msg_len = 10485760
```
Comment (hicqu) [inline etc/config-template.toml:150]: s/_/-/g

**Comment by hicqu** [comment]
Rest LGTM. Thank you!

**Review by tabokie — APPROVED** [review]

**Review by hicqu — APPROVED** [review]

**Pull request merged**